api/spec/v1.yaml

```diff
@@ -594,6 +594,27 @@ paths:
                 $ref: '#/components/schemas/DelegationList'
         <<: *common_error_responses
 
+  /consensus/accounts/{address}/delegations_to:
```

Comment:
Used `(debonding)delegations_to` for the method and API names. This is the same terminology as used in oasis-core. 

I didn't like using `delegators` because that would imply (at least to me) that the endpoint only returns the delegators (just a list of addresses) to an address. However, the endpoint returns the full delegations (delegator-delegatee pair, and amount), same as the `delegations` API counterpart.

Reply:
Excellent, I had the same misgiving about the proposed `delegators` endpoint name, and the same idea/wish for the `Delegation` struct.

I was also hoping we'd unify delegations and debonding delegations more :), but realistically the hassle/benefit ratio is probably not worth it. Thank you!

Reply:
> I was also hoping we'd unify delegations and debonding delegations more :), but realistically the hassle/benefit ratio is probably not worth it. Thank you!

Maybe it would make sense unifying the debonding delegations with delegations more, but I'm not super sure about that.

It's not only that the debonding delegations are delegations with an end period, the debonding delegations also have a semantic difference - the delegator doesn't earn staking rewards for those shares, so these are actually "locked" shares.

So:
- delegations -> escrowed shares
- debonding delegations -> locked shares

So in some sense it makes sense to me to have these separated.

---

However, maybe with less in-depth logic of oasis-core (which is likely the majority of consumers of the API) this doesn't make all that much sense and it would be clearer to just merge these two. No strong opinions about it tbh.